frontend/context/StakingProgramProvider.tsx

```diff
@@ -33,8 +40,7 @@ const useGetActiveStakingProgramId = (serviceNftTokenId: Optional<number>) => {
   const response = useQuery({
     queryKey: REACT_QUERY_KEYS.STAKING_PROGRAM_KEY(evmHomeChainId),
     queryFn: async () => {
-      if (!serviceNftTokenId)
-        return DEFAULT_STAKING_PROGRAM_IDS[selectedAgentConfig.evmHomeChainId];
+      if (!serviceNftTokenId) return null;
```

Comment:
this should be null when there's no service deployed yet, otherwise there are problems with other staking programs alerts, can discuss deeply if needed